webpack/assets/javascripts/react_app/redux/consts.js

```diff
@@ -0,0 +1,3 @@
+export const STATISTICS_DATA_REQUEST = 'STATISTICS_DATA_REQUEST';
+export const STATISTICS_DATA_SUCCESS = 'STATISTICS_DATA_SUCCESS';
```

Comment:
I have a bias towards using only one constants file. Also, prefer a name that doesn't have a spelling error.